src/libraries/Microsoft.Extensions.Logging/src/LoggerInformation.cs

```diff
@@ -19,7 +19,7 @@ public MessageLogger(ILogger logger, string? category, string? providerTypeFullN
 
         public ILogger Logger { get; }
 
-        public string? Category { get; }
```

Comment:
>public string? Category { get; } [](http://example.com/codeflow?start=8&length=32)

are we sure the category never be null?

Reply:
Nullable analysis says it can't. I changed it because I didn't see any code paths that allowed null.